lib/gis/asprintf.c

```diff
@@ -43,23 +43,21 @@ int G_vasprintf(char **out, const char *fmt, va_list ap)
 #ifdef HAVE_ASPRINTF
     return vasprintf(out, fmt, ap);
 #else
+    va_list aq;
```

Comment:
Maybe not a big deal at this point, but I thought this was in some other PR.

Reply:
You're right. Not sure how that's here.

Reply:
[That PR](https://github.com/OSGeo/grass/pull/3650) is not merged yet and this change is reverted.